src/plugins/custom_integrations/server/language_clients/index.ts

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { CoreSetup } from 'kibana/server';
+import { CustomIntegrationRegistry } from '../custom_integration_registry';
+import { CustomIntegrationIcon, PLUGIN_ID } from '../../common';
+
+interface LanguageIntegration {
+  id: string;
+  title: string;
+  icon?: string;
+  euiIconName?: string;
+  description: string;
+  docUrl: string;
+}
+
+export const integrations: LanguageIntegration[] = [
+  {
+    id: 'all',
+    title: i18n.translate('customIntegrations.languageclients.AllTitle', {
+      defaultMessage: 'Elasticsearch Clients',
+    }),
+    euiIconName: 'logoElasticsearch',
+    description: i18n.translate('customIntegrations.languageclients.AllDescription', {
+      defaultMessage:
+        'Start building your custom application on top of Elasticsearch with the official language clients.',
+    }),
+    docUrl: 'https://www.elastic.co/guide/en/elasticsearch/client/index.html',
```

Comment:
@lcawl From what I understand, this is not the preferred approach to link to docs. Do you have some examples on how these links should be constructed instead?

Reply:
On the front-end we have a `core.docLinks` API for this purpose, but there's no server-side analog. I think would be ok to follow a similar pattern though to build the links with the appropriate branch specifier by pulling the branch from `core.initContext.env.kibanaBranch` and then constructing the links similar to the client-side API:

https://github.com/elastic/kibana/blob/6bfa2a4c2ce4a32fa6857cfb5b3393b2cf89da09/src/core/public/doc_links/doc_links_service.ts#L22-L27

Reply:
thx. plz see last change for this approach.

Reply:
One of the nice benefits of having all the front-end links derive from that doc link service is that we can check for breakages as part of our doc builds (https://github.com/elastic/docs/blob/master/build_docs.pl#L344). And when there are significant changes to URLs (e.g. new docs system), we only have to change one file. If there are going to be a lot of server-side links, it would be nice to have an equivalent single source file that we could potentially add to our checks too.  If that's not possible imminently, perhaps we can add these URLs to the existing doc link service (so they're tested in the mean time) and add a comment to remind folks to clean them up in this other index.ts file too.